src/app/modules/app/pages/profile/profile.component.html

```diff
@@ -1 +1,25 @@
-<p>profile works!</p>
+<div class="profile-container">
+  <button mat-icon-button class="back-button" (click)="goBack()">
+    <mat-icon>arrow_back</mat-icon>
+  </button>
```

Comment:
It should be good to have the 'Profile' title next to the arrow back icon, so the user can explicitly know which page they are on.